README.md
# [Argon React Native](https://creativetimofficial.github.io/argon-react-native/docs/#) [![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social&logo=twitter)](https://twitter.com/home?status=Argon%20React%20Native,%20a%20cool%20Argon%20React%20Native%20App%20Template%20%E2%9D%A4%EF%B8%8F%20https%3A//bit.ly/2KAj86H%20%23reactnative%20%23argon%20%23designsystem%20%23developers%20via%20%40CreativeTim)


 ![version](https://img.shields.io/badge/version-1.7.0-blue.svg)  [![GitHub issues open](https://img.shields.io/github/issues/creativetimofficial/argon-react-native.svg?style=flat)](https://github.com/creativetimofficial/argon-react-native/issues?q=is%3Aopen+is%3Aissue) [![GitHub issues closed](https://img.shields.io/github/issues-closed-raw/creativetimofficial/argon-react-native.svg?maxAge=2592000)](https://github.com/creativetimofficial/argon-react-native/issues?q=is%3Aissue+is%3Aclosed)


![Product Gif](https://raw.githubusercontent.com/creativetimofficial/public-assets/master/argon-react-native/arg-rn-thumbnail.jpg)

Argon React Native is a fully coded app template built over [Galio.io](https://galio.io/?ref=creativetim), [React Native](https://facebook.github.io/react-native/?ref=creativetim) and [Expo](https://expo.io/?ref=creativetim) to allow you to create powerful and beautiful e-commerce mobile applications. We have redesigned all the usual components in Galio to make it look like Argon's Design System, minimalistic and easy to use.

Start your development with a badass Design System for React Native inspired by Argon Design System. If you like Argon's Design System, you will love this react native app template! It features a huge number of components and screens built to fit together and look amazing. 

### FULLY CODED COMPONENTS

Argon React Native features over 200 variations of components like buttons, inputs, cards, navigations etc, giving you the freedom of choosing and combining. All components can take variations in colour, that you can easily modify inside our theme file.

You will save a lot of time going from prototyping to full-functional code, because all elements are implemented. We wanted the design process to be seamless, so switching from image to the real page is very easy to do.

### Components & Cards
Argon React Native comes packed with a large number of components and cards. Putting together a mobile app has never been easier than matching together different components. From the profile screen to a settings screen, you can easily customise and build your screens. We have created multiple options for you to put together and customise into pixel perfect screens. 

View [ all components/cards here](https://demos.creative-tim.com/argon-react-native/index.html#cards).

### Example Screens
If you want to get inspiration or just show something directly to your clients, you can jump start your development with our pre-built example screens. From onboarding screens to profile or discover screens, you will be able to quickly set up the basic structure for your React Native mobile project. 

View [all screens here](https://demos.creative-tim.com/argon-react-native/index.html#screens).


Let us know your thoughts below. And good luck with development!


## Table of Contents

* [Versions](#versions) 
* [Demo](#demo)
* [Quick Start](#quick-start)
* [Documentation](#documentation)
* [File Structure](#file-structure)
* [OS Support](#os-support)
* [Resources](#resources)
* [Reporting Issues](#reporting-issues)
* [Technical Support or Questions](#technical-support-or-questions)
* [Licensing](#licensing)
* [Useful Links](#useful-links)

## Versions

[<img src="https://github.com/creativetimofficial/public-assets/blob/master/logos/html-logo.jpg?raw=true" width="60" height="60" />](https://www.creative-tim.com/product/argon-design-system)[<img src="https://github.com/creativetimofficial/public-assets/blob/master/logos/vue-logo.jpg?raw=true" width="60" height="60" />](https://www.creative-tim.com/product/vue-argon-design-system)[<img src="https://github.com/creativetimofficial/public-assets/blob/master/logos/react-logo.jpg?raw=true" width="60" height="60" />](https://www.creative-tim.com/product/argon-design-system-react)[<img src="https://github.com/creativetimofficial/public-assets/blob/master/logos/react-native-logo.jpg?raw=true" width="60" height="60" />](https://www.creative-tim.com/product/argon-react-native)[<img src="https://github.com/creativetimofficial/public-assets/blob/master/logos/angular-logo.jpg?raw=true" width="60" height="60" />](https://www.creative-tim.com/product/argon-dashboard-angular)





| HTML | React | Angular  |
| --- | --- | ---  |
| [![Argon Design System](https://raw.githubusercontent.com/creativetimofficial/public-assets/master/argon-design-system/argon-design-system.jpg)](https://www.creative-tim.com/product/argon-design-system)  | [![Argon Design System React](https://raw.githubusercontent.com/creativetimofficial/public-assets/master/argon-design-system-react/argon-design-system-react.jpg)](https://www.creative-tim.com/product/argon-design-system-react)  | [![Argon Design System Angular](https://raw.githubusercontent.com/creativetimofficial/public-assets/master/argon-design-system-angular/argon-design-system-angular.jpg)](https://www.creative-tim.com/product/argon-design-system-angular)

## Demo

| Home Screen | Profile Screen | Elements Screen | Register Screen |
| --- | --- | --- | --- |
| [![Home Screen](https://raw.githubusercontent.com/creativetimofficial/public-assets/master/argon-react-native/home-screen.png)](https://demos.creative-tim.com/argon-react-native/) | [![Profile Screen](https://raw.githubusercontent.com/creativetimofficial/public-assets/master/argon-react-native/profile-screen.png)](https://demos.creative-tim.com/argon-react-native/) | [![Elements Screen](https://raw.githubusercontent.com/creativetimofficial/public-assets/master/argon-react-native/elements-screen.png)](https://demos.creative-tim.com/argon-react-native/) | [![Register Screen](https://raw.githubusercontent.com/creativetimofficial/public-assets/master/argon-react-native/register-screen.png)](https://demos.creative-tim.com/argon-react-native/) |

- [Start page](https://demos.creative-tim.com/argon-react-native)
- [How to install our free demo](https://demos.creative-tim.com/argon-react-native/docs/#/install)

[View more](https://demos.creative-tim.com/argon-react-native)

## Quick start
- Try it out on Expo (Simulator for iOS or even your physical device if you have an Android)
- Download from [Creative Tim](https://www.creative-tim.com/product/argon-react-native)


## Documentation
The documentation for the Argon React Native is hosted at our [website](https://demos.creative-tim.com/argon-react-native/docs/).


## File Structure
Within the download you'll find the following directories and files:

```
argon-react-native/
├── App.js
├── README.md
├── app.json
├── assets
├── babel.config.js
├── components
│   ├── Button.js
│   ├── DrawerItem.js
│   ├── Header.js
│   ├── Icon.js
│   ├── Card.js
│   ├── Select.js
│   ├── Switch.js
│   ├── Tabs.js
│   └── index.js
├── constants
│   ├── Images.js
│   ├── Theme.js
│   ├── index.js
│   ├── articles.js
│   ├── tabs.js
│   └── utils.js
├── navigation
│   ├── Menu.js
│   └── Screens.js
├── package.json
├── screens
│   ├── Articles.js
│   ├── Home.js
│   ├── Elements.js
│   ├── Onboarding.js
│   ├── Pro.js
│   ├── Profile.js
│   └── Register.js

```


## OS Support

At present, we officially aim to support the last two versions of the following operating systems:

[<img src="https://raw.githubusercontent.com/creativetimofficial/ct-material-kit-pro-react-native/master/assets/android-logo.png" width="60" height="60" />](https://www.creative-tim.com/product/material-kit-pro-react-native)[<img src="https://raw.githubusercontent.com/creativetimofficial/ct-material-kit-pro-react-native/master/assets/apple-logo.png" width="60" height="60" />](https://www.creative-tim.com/product/material-kit-pro-react-native)



## Resources
- Demo: <https://demos.creative-tim.com/argon-react-native>
- Download Page: <https://www.creative-tim.com/product/argon-react-native>
- Documentation: <https://demos.creative-tim.com/argon-react-native/docs>
- License Agreement: <https://www.creative-tim.com/license>
- Support: <https://www.creative-tim.com/contact-us>
- Issues: [Github Issues Page](https://github.com/creativetimofficial/argon-react-native/issues)
- [Argon Design System](https://www.creative-tim.com/product/argon-design-system?ref=argonrn-readme) - For Front End Development
- **Dashboards:**

| HTML | React | Angular |
| --- | --- | ---  |
| [![Argon HTML](https://raw.githubusercontent.com/creativetimofficial/public-assets/master/argon-dashboard/argon-dashboard.jpg)](https://www.creative-tim.com/product/argon-dashboard) | [![Argon Dashboard React](https://raw.githubusercontent.com/creativetimofficial/public-assets/master/argon-dashboard-react/argon-dashboard-react.jpg)](https://www.creative-tim.com/product/argon-dashboard-react) | [![Argon Dashboard Angular](https://raw.githubusercontent.com/creativetimofficial/public-assets/master/argon-dashboard-angular/argon-dashboard-angular.jpg)](https://www.creative-tim.com/product/argon-dashboard-angular)

| Node.js | ASP.NET  |
| --- | --- |
| [![Argon Dashboard NodeJS](https://raw.githubusercontent.com/creativetimofficial/public-assets/master/argon-dashboard-nodejs/argon-dashboard-nodejs.jpg)](https://www.creative-tim.com/product/argon-dashboard-nodejs) | [![Argon Dashboard ASP.NET](https://github.com/creativetimofficial/public-assets/blob/master/argon-dashboard-aspnet/argon-dashboard-aspnet.gif)](https://www.creative-tim.com/product/argon-dashboard-asp-net)


## Reporting Issues

We use GitHub Issues as the official bug tracker for the Argon React Native. Here are some advices for our users that want to report an issue:

1. Make sure that you are using the latest version of the Argon React Native.
2. Providing us reproducible steps for the issue will shorten the time it takes for it to be fixed.
3. Some issues may be browser specific, so specifying in what browser you encountered the issue might help.


### Technical Support or Questions

If you have questions or need help integrating the product please [contact us](https://www.creative-tim.com/contact-us) instead of opening an issue.


## Licensing

- Copyright 2020 Creative Tim (https://www.creative-tim.com/)

- Licensed under MIT (https://github.com/creativetimofficial/argon-react-native/blob/master/LICENSE.md)



## Useful Links

- [Tutorials](https://www.youtube.com/channel/UCVyTG4sCw-rOvB9oHkzZD1w)
- [Affiliate Program](https://www.creative-tim.com/affiliates/new) (earn money)
- [Blog Creative Tim](http://blog.creative-tim.com/)
- [Free Products](https://www.creative-tim.com/bootstrap-themes/free) from Creative Tim
- [Premium Products](https://www.creative-tim.com/bootstrap-themes/premium) from Creative Tim
- [React Products](https://www.creative-tim.com/bootstrap-themes/react-themes) from Creative Tim
- [Angular Products](https://www.creative-tim.com/bootstrap-themes/angular-themes) from Creative Tim
- [VueJS Products](https://www.creative-tim.com/bootstrap-themes/vuejs-themes) from Creative Tim
- [More products](https://www.creative-tim.com/bootstrap-themes) from Creative Tim
- Check our Bundles [here](https://www.creative-tim.com/bundles?ref="argon-github-readme")


### Social Media

Twitter: <https://twitter.com/CreativeTim>

Facebook: <https://www.facebook.com/CreativeTim>

Dribbble: <https://dribbble.com/creativetim>

Instagram: <https://www.instagram.com/CreativeTimOfficial>



CHANGELOG.md
## [1.7.0] 2020 - 11 - 04
### Updated dependencies
- updated `@react-navigation/bottom-tabs@5.06` to `@react-navigation/bottom-tabs@5.5.1`
- updated `@react-navigation/compat@5.0.0` to `@react-navigation/compat@5.1.25`
- updated `@react-navigation/drawer@5.0.0` to `@react-navigation/drawer@5.12.4`
- updated `@react-navigation/native@5.0.0` to `@react-navigation/native@5.9.3`
- updated `@react-navigation/stack@5.0.0` to `@react-navigation/stack@5.14.3`
- updated `expo SDK@39` to `expo SDK@40`
- updated `expo-asset@8.2.0` to `expo-asset@8.2.1`
- updated `expo-font@8.3.0` to `expo-font@8.4.0`
- updated `react-native SDK@39` to `react-native SDK@40`
- updated `react-native-gesture-handler@1.7.0` to `react-native-gesture-handler@1.8.0`
- updated `react-native-safe-area-context@3.1.4` to `react-native-safe-area-context@3.1.9`
- updated `react-native-screens@2.10.1` to `react-native-screens@2.15.2`
- added `expo-app-loading@1.01`

### Updated files
- `useNativeDriver` warning fixed in `Tabs.js`
- card height changed in `Register.js`


## [1.6.0] 2020 - 11 - 04
### Updated dependencies
- updated `expo-asset@8.1.5` to `expo-asset@8.2.0`
- updated `expo-font@8.1.0` to `expo-font@8.3.0`
- updated `react-native-gesture-handler@1.6.0` to `react-native-gesture-handler@1.7.0`
- updated `react-native-reanimated@1.7.0` to `react-native-reanimated@1.13.0`
- updated `react-native-screens@2.2.0` to `react-native-screens@2.10.1`
- updated `react-native-safe-area-context@0.7.3` to `react-native-safe-area-context@3.1.4`
- updated `@react-native-community/maksed-view@0.1.6` to `@react-native-community/maksed-view@0.1.10`
- updated `react-native SDK@37.0.0` to `react-native SDK@39.0.0`
- updated `react@16.9.0` to `react@16.13.1`
- updated `babel-preset-expo@8.2.1` to `babel-preset-expo@8.3.0`
- updated `expo SDK@37.0.0` to `expo SDK@39.0.0`
- updated `galio-framework@0.6.3` to `galio-framework@0.7.1`
- changed the git source for react-native-modal-dropdown

### Updated files
- Profile.js - fixed elements regarding the Photo Album
- Elements.js - ScrollView fixed by adding width, PR accepted which removed a duplicate styling [#24](https://github.com/creativetimofficial/argon-react-native/pull/24)
- App.js - fixed Invariant Violation via PR [#29](https://github.com/creativetimofficial/argon-react-native/pull/29)

## [1.5.0] 2020 - 06 - 04
### Updated dependencies
- updated `expo-asset@8.0.0` to `expo-asset@8.1.5`
- updated `expo-font@8.0.0` to `expo-font@8.1.0`
- updated `react-native-gesture-handler@1.5.0` to  `react-native-gesture-handler@1.6.0`
- updated `react-native-reanimated@1.4.0` to `react-native-reanimated@1.7.0`
- updated `react-native-screens@2.0.0-alpha.12` to `react-native-screens@2.2.0`
- updated `react-native-safe-area-context@0.6.0` to `react-native-safe-area-context@0.7.3`
- updated `@react-native-community/masked-view@0.1.5` to `@react-native-community/masked-view@0.1.6`
- updated `react-native SDK@36.0.0` to `react-native SDK@37.0.0`
- updated `babel-preset-expo@7.0.0` to `babel-preset-expo@8.2.1`
- updated `Expo @36.0.0` to `Expo @37.0.0`

### Updated files
- used hooks for App.js
- moved `assets/font/argon.json` to `assets/config/argon.json` in order to make sure there wouldn't be any issue with the build for Android

## [1.4.0] 2020 - 03 - 05
### Removed dependencies
- removed `react-navigation@3.11.0`

### Added dependencies
- added `@react-navigation/compat@5.0.0`
- added `@react-navigation/drawer@5.0.0`
- added `@react-navigation/native@5.0.0`
- added `@react-navigation/stack@5.0.0`
- added `@react-native-community/masked-view@0.1.5`
- added `react-native-reanimated@1.4.0`
- added `react-native-safe-area-context@0.6.0`
- added `react-native-screeens@2.0.0-alpha.12`

### Updated dependencies
- updated `expo@35.0.0` to `expo@36.0.0`
- updated `expo-asset@7.0.0` to `expo-asset@8.0.0`
- updated `expo-font@7.0.0` to `expo-font@8.0.0`
- updated `expo-cli@2.4.0` to `expo-cli@3.11.7`
- updated `expo-linear-gradient@7.0.0` to `expo-linear-gradient@8.0.0`
- updated `react@16.8.3` to `react@16.9.0`

### Updated files
- changed the whole routing from `Screens.js` because `react-navigation@5.0.0` has a new dynamic API
- changed `Menu.js` for a new Drawer custom component
- changed `DrawerItem.js` for a new type of `<DrawerCustomItem />`
- changed props and variables so that the new `react-navigation` API could work with the following files: `Beauty.js`, `Header.js`, `Product.js`, `Gallery.js`, `Pro.js`, `Product.js`, `Settings.js`, `Register.js`, `Onboarding.js`

## [1.3.0] 2019-11-06
### Updated dependencies
- `expo@34.0.3` to `expo@35.0.0`
- `expo-font@6.0.1` to `expo-font@7.0.0`
- `expo-asset@6.0.0` to `expo-asset@7.0.0`
- `react-native SDK@34.0.0` to `react-native SDK@35.0.0`
- `galio-framework@0.6.1` to `galio-framework@0.6.3`
- `babel-preset-expo@5.0.0` to `babel-preset-expo@7.0.0`

### Updated files
- changed drawer items in order to have a similar look to the PRO version

## [1.2.0] 2019-09-18
### Updated dependencies
- `expo@33.0.0` to `expo@34.0.3`
- `expo-font@5.0.1` to `expo-font@6.0.1`
- added `expo-asset@6.0.0`
- `react-native SDK@33.0.0` to `react-native SDK@34.0.0`
- added `react-native-gesture-handler@1.3.0`
- `galio-framework@0.5.3` to `galio-framework@0.6.1`

## [1.1.0] 2019-06-21
### Updated dependencies
- `expo@32.0.0` to `expo@33.0.0`
- `galio-framework@0.5.1` to `galio-framework@0.5.3`
- `react-native SDK@32.0.0` to `react-native SDK@33.0.0`
- `react-navigation@3.8.1` to `react-navigation@3.11.0`
- `react@16.5.0` to `react@16.8.3`
- added `expo-font@5.0.1`


## [1.0.0] 2019-06-07
### Initial Release


screens/Profile.js
import React, { useEffect, useState } from "react";
import {
  StyleSheet,
  Dimensions,
  ScrollView,
  Image,
  TouchableOpacity
} from "react-native";
import { Block, Text, theme } from "galio-framework";
import { Icon } from '../components';
import { Button } from "../components";
import { argonTheme } from "../constants/index";
import ArticleCard from '../components/ArticleCard';
import AsyncStorage from "@react-native-async-storage/async-storage";
//import { TouchableOpacity } from "react-native-gesture-handler";

const perfil = require("../assets/imgs/fotoPerfil.jpg");

const { width, height } = Dimensions.get("screen");
const { getRegistrosByCliente } = require("../services/registroDeSubasta.service");
const { getFotosByProducto } = require("../services/foto.service");
const { getProductoById } = require("../services/producto.service");
const { getProductosByCliente } = require("../services/producto.service");
const { getPersona } = require("../services/persona.service");


const thumbMeasure = (width - 48 - 32) / 3;

const Profile = () => {

  const [misArticulos, setMisArticulos] = useState([]);
  const [categoria, setCategoria] = useState();
  const [direccion, setDireccion] = useState();
  const [nombre, setNombre] = useState();
  const [cliente, setCliente] = useState();
  const [subastasParticipadas, setSubastasParticipadas] = useState(0);
  const [articulosSubastados, setArticulosSubastados] = useState(0);
  const [objetos,setObjetos] = useState();
  useEffect(() => {
    recuperar();
    getRegistrosCliente(); 
    lenProductosSubastados();
  }, [])

  const recuperar = async () => { 
    try {
      const categoriaCliente = await AsyncStorage.getItem('categoria');
      const idCliente = await AsyncStorage.getItem('idCliente');
      const res = await getPersona(idCliente);
      //console.log(res);
      setCliente(idCliente);
      setCategoria(categoriaCliente);
      setNombre(res.persona.nombre);
      setDireccion(res.persona.direccion);
    } catch (error) {
        console.log(error);
    }
  }

  const lenProductosSubastados = async () => {
    const idCliente = await AsyncStorage.getItem('idCliente');
    const { productos } = await getProductosByCliente(idCliente);
    setArticulosSubastados(productos.length)
  }

  const getRegistrosCliente = async () => {
    const idCliente = await AsyncStorage.getItem('idCliente');
    const idProductos = [];
    const idObjetos = [];
    const { listaPujasDeCliente } = await getRegistrosByCliente(idCliente);
    for (const registro in listaPujasDeCliente) {
      if (!idProductos.includes(listaPujasDeCliente[registro].producto)) {
        idProductos.push(listaPujasDeCliente[registro].producto)
        idObjetos.push({idProducto:listaPujasDeCliente[registro].producto,idSubasta:listaPujasDeCliente[registro].subasta})
      };
    }
    setObjetos(idObjetos)
    const prodPromiseArr = [];
    console.log(idProductos)
    idProductos.forEach(async idProducto => {
      
      prodPromiseArr.push(getProductoById(idProducto));
    });
    const products = await Promise.all(prodPromiseArr);
    const fotoPromiseArr = [];
    products.forEach(producto => {
      fotoPromiseArr.push(getFotosByProducto(producto.producto.idProducto));
    });
    const fotos = await Promise.all(fotoPromiseArr);
    const productArray = [];
    fotos.forEach(foto => {
      const res = products.filter(product => product.producto.idProducto === foto.fotos[0].idProducto);
      res[0].producto.foto = foto.fotos[0].referencia_url;
      productArray.push(res[0]);
    });
    setMisArticulos(productArray);
    setSubastasParticipadas(productArray.length);
  }

  return (
    <Block flex style={styles.profile}>
      <Block flex>
        <Block
          //zsource={Images.ProfileBackground}
          style={styles.profileContainer, { backgroundColor: '#EEBB00' }}
          imageStyle={styles.profileBackground}
        >
          <ScrollView
            showsVerticalScrollIndicator={false}
            style={{ width, marginTop: '22%' }}
          >
            <Block flex style={styles.profileCard}>
              <Block middle style={styles.avatarContainer}>
                <Image
                  source={perfil}
                  style={styles.avatar}
                />
              </Block>
              <Block style={styles.info}>
                <Block style={{ marginTop: 15 }} row space="evenly">
                  <Block middle>
                    <Text
                      bold
                      size={18}
                      color="#525F7F"
                      style={{ marginBottom: 4 }}
                    >
                      -
                      </Text>
                    <Block>
                      <Text style={{ textAlign: 'center' }} size={12} color={argonTheme.COLORS.TEXT}>Subastas</Text>
                      <Text style={{ textAlign: 'center' }} size={12} color={argonTheme.COLORS.TEXT}>ganadas</Text>
                    </Block>
                  </Block>
                  <Block middle>
                    <Text
                      bold
                      color="#525F7F"
                      size={18}
                      style={{ marginBottom: 4 }}
                    >
                      {subastasParticipadas}
                      </Text>
                    <Block>
                      <Text style={{ textAlign: 'center' }} size={12} color={argonTheme.COLORS.TEXT}>Subastas</Text>
                      <Text style={{ textAlign: 'center' }} size={12} color={argonTheme.COLORS.TEXT}>participadas</Text>
                    </Block>
                  </Block>
                  <Block middle>
                    <Text
                      bold
                      color="#525F7F"
                      size={18}
                      style={{ marginBottom: 4 }}
                    >
                      {articulosSubastados}
                      </Text>
                    <Block>
                      <Text style={{ textAlign: 'center' }} size={12} color={argonTheme.COLORS.TEXT}>Artículos</Text>
                      <Text style={{ textAlign: 'center' }} size={12} color={argonTheme.COLORS.TEXT}>subastados</Text>
                    </Block>
                  </Block>
                </Block>
              </Block>
              <Block flex>
                <Block middle style={styles.nameInfo}>
                  <Text bold size={28} color="#32325D">
                    {nombre}
                  </Text>
                  <Text size={18} color="#32325D" style={{ marginTop: 8 }}>
                    {direccion}
                  </Text>
                  <Block row style={{ marginTop: 2 }}>
                    <Icon
                      name={"Trophy"} family="AntDesign"
                      size={16}
                      style={{ marginTop: 5, marginHorizontal: 3 }}
                      color='brown'
                    />
                    <Text size={16} color="#32325D" style={{ marginTop: 2 }}>
                      {categoria}
                    </Text>
                  </Block>
                </Block>
                <Block middle style={{ marginTop: 20, marginBottom: 16 }}>
                  <Block style={styles.divider} />
                </Block>
                <Block middle>
                  {misArticulos.map((articulo,index) => {
                    
                    return (
                       
                        <ArticleCard key={articulo.producto.idProducto} item={objetos[index]} perfil={true} estadoFinal={'sdsd'} imagen={{ uri: articulo.producto.foto }} titulo={articulo.producto.descripcion} estado={'aprobado'} horizontal />
                      

                    );
                  })}
                
                </Block>

              </Block>
            </Block>
          </ScrollView>
        </Block>
      </Block>
    </Block>
  );
}

const styles = StyleSheet.create({
  profile: {
    marginTop: 0,
    //marginTop: Platform.OS === "android" ? -HeaderHeight : 0,
    // marginBottom: -HeaderHeight * 2,
    flex: 1
  },
  profileContainer: {
    width: width,
    height: height,
    padding: 0,
    zIndex: 1
  },
  profileBackground: {
    width: width,
    height: height / 2
  },
  profileCard: {
    // position: "relative",
    padding: theme.SIZES.BASE,
    marginHorizontal: theme.SIZES.BASE,
    marginTop: 65,
    borderTopLeftRadius: 6,
    borderTopRightRadius: 6,
    backgroundColor: theme.COLORS.WHITE,
    shadowColor: "black",
    shadowOffset: { width: 0, height: 0 },
    shadowRadius: 8,
    shadowOpacity: 0.2,
    zIndex: 2
  },
  info: {
    paddingHorizontal: 40
  },
  avatarContainer: {
    position: "relative",
    marginTop: -80
  },
  avatar: {
    width: 124,
    height: 124,
    borderRadius: 62,
    borderWidth: 0
  },
  nameInfo: {
    marginTop: 35
  },
  divider: {
    width: "90%",
    borderWidth: 1,
    borderColor: "#E9ECEF"
  },
  thumb: {
    borderRadius: 4,
    marginVertical: 4,
    alignSelf: "center",
    width: thumbMeasure,
    height: thumbMeasure
  }
});

export default Profile;
